spec/donation_system/salesforce/donation_validator_spec.rb

```diff
@@ -37,6 +37,50 @@ module Salesforce
         end
       end
 
+      describe 'optional fields' do
+        it 'can have a currency' do
```

Comment:
This type of test gets repeated a lot now. Is it worth adding a custom matcher so that the tests aren't so coupled to `fields` being a hash?

Reply:
I'll think about this as well, I want to have a balance between being expressive and not scaring beginners / first time pull-requesters